make/RunTests.gmk

```diff
@@ -702,6 +703,58 @@ define SetJtregValue
   endif
 endef
 
+
```

Comment:
This PR was not tagged with the `build` label, and was therefore not brought to the attention of the build team.

This file contains a large number of unacceptable changes. :-( It would under no circumstance have been approved as it looks. 

Please make sure that the Github labels actually matches the set of files that are changed in the PR. Skara tooling will assist in this, but the ultimate responsibility lies with the Author and Reviewers of the PR.